.gitignore

```diff
@@ -5,3 +5,4 @@ lsp.log
 pkg*
 .vscode
 Cargo.lock
+.idea
```

Comment:
Why was this necessary? Is this some file created by your IDE?

Reply:
Yeah these are files created by GoLand

Reply:
It is in the flux .gitignore as well.